pactsafe/pactsafe-react-sdk
# Add customData and acceptanceLanguage Props

## Description
This brings the ability to add custom data to acceptance events with a new `customData` prop. Additionally, you can now set the acceptance language (using a new `acceptanceLanguage` prop) that gets displayed as part of your clickwrap group. Note: setting acceptance language requires the group to be dynamic.

## Testing
Unit tests should cover making sure the props get set correctly. But, to test locally, you should:
- Set the `customData` to an object (e.g., `{key_1: "test", key_2: 3}`) and trigger an acceptance. With this acceptance, if you go to the Clickwrap UI and view the acceptance event, you should see the custom data details in the right panel.
- Set the `acceptanceLanguage` prop to a string and load the clickwrap in a test environment. In the loaded environment, you should see the acceptance language be presented.
